### PR TITLE
Remove python2 pip from docker images

### LINKED ISSUE #5976: Weekly docker image fails to build
Usual build command: `docker build --no-cache -t owasp/zap2docker-weekly -f docker/Dockerfile-weekly docker/`
Worked before, now failing with
```
Reading package lists...
Reading package lists...
Building dependency tree...
Reading state information...
E: Unable to locate package python-pip
```

### PR DESCRIPTION
**Pull request by psiinon**
Fixes #5976

Signed-off-by: Simon Bennetts <psiinon@gmail.com>

### CODE REVIEW

**Comment by thc202** [comment]
Could you update the changelog to mention that the Python 2 zaproxy package will no longer be available?

**Comment by psiinon** [comment]
Done.

**Comment by thc202** [comment]
Thanks!

File: docker/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2020-05-12
+ - Removed python 2, only python 3 will be supported going forwards
```
Comment (kingthorin) [inline docker/CHANGELOG.md:5]:
```suggestion
 - Removed python 2, only python 3 will be supported going forward.
```

**Comment by psiinon** [comment]
Done

**Pull request merged**